.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "SelectedNode": "\\C:\\Users\\Dsvk2\\Source\\Repos\\opensource.guide",
```

Comment:
I imagine we're not going to want this line as it is specific to your user.  Correct @BlueButterflies?  Do we want this file committed? /cc @ahpook

Reply:
nope, this shouldn't be added at all

Reply:
Thank you for report. I didn't see that I uploaded the .vs folder. I've already solved it, so you can proceed with the review